akka-http-caching/src/main/scala/akka/http/caching/javadsl/CachingSettings.scala

```diff
@@ -6,9 +6,8 @@ package akka.http.caching.javadsl
 
```

Comment:
Directory structure is not corresponding to that of impl classes (`impl.settings.CacheSettingsImpl`), because {javadsl, scaladsl}.CachingSettings object has already been used by other parts. 
Assuming changing package `javadsl` to `javadsl.settings` may cause application codes that instantiate CacheSettings to not function anymore, I let this structure remained.

Reply:
Yes, for this small module, it's fine.